Lib/multiprocessing/connection.py

```diff
@@ -203,7 +202,7 @@ def send(self, obj):
         """Send a (picklable) object"""
         self._check_closed()
         self._check_writable()
-        self._send_bytes(_ForkingPickler.dumps(obj))
+        self._send_bytes(context.reduction.ForkingPickler.dumps(obj))
```

Comment:
Note that `pickle.dumps` and `pickle.loads` are short convenience wrappers around `Pickler.dump` and `Unpickler.load`, respectively (see `Lib/pickle.py`). So we can redefine our own `dumps` and `loads` helpers in the same way.

Reply:
:+1: